libraries/AP_HAL_ChibiOS/hwdef/BeastH7v2/hwdef.dat

```diff
@@ -0,0 +1,42 @@
+# hw definition file for processing by chibios_pins.py
+# for iFlight Beast H7 v2 hardware, based on Beast H7 v1
+# thanks to betaflight for pin information
+
+include ../BeastH7/hwdef.dat
+undef APJ_BOARD_ID
+undef PD15 PB10 PB11 
+undef HAL_SERIAL3_PROTOCOL
```

Comment:
You need to define serial2 protocol as a default to 23 since that is defined as the default serial RX uart. You also need to disable the usart2 TX pin since that is not available.

Reply:
@andyp1per , TX2 exists on the back side of board. It also has printed marks on it. I have tested RX2/TX2 with connecting UART-USB converter and using Mavlink2 protocol with Mission Planner. I have updated the pictures of the pinout to show that. Serial2 with default 23 made in today's commit